### ISSUE TITLE
apng dose not play

### ISSUE THREAD

**Issue by ffts** [penfeizhou/APNG4Android#71]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |   G8142   |
| System Version               | e.g.  6.0                                             |   9.0   |
| APNG4Android Library Version | e.g.  1.0.0                                           |   2.4.0   |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |   100%   |
| Repro with our demo project  | e.g. does it happen with our demo project?            |  yes    |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Enter the APNGTestActivity and the images didn't play.

剩下的还是说中文吧...
看了下应该是GlideAnimationModule里的这几行导致的
```
AnimationFrameDecoder animationFrameDecoder = new AnimationFrameDecoder(glide.getBitmapPool());
registry.prepend(ByteBuffer.class, Bitmap.class, animationFrameDecoder);
registry.prepend(InputStream.class, Bitmap.class, new StreamFrameDecoder(animationFrameDecoder));
```
加了之后glide找decoder的时候，DecodePath里就没有ByteBufferAnimationDecoder和StreamAnimationDecoder了
<img width="348" alt="屏幕快照 2020-08-21 下午3 56 59" src="https://user-images.githubusercontent.com/1276568/90868019-d1244180-e3c8-11ea-81b3-5085302e060c.png">

注释掉这几行的话
<img width="361" alt="屏幕快照 2020-08-21 下午3 55 49" src="https://user-images.githubusercontent.com/1276568/90868049-daada980-e3c8-11ea-859e-94dd0e74c0f7.png">
然后apng也可以动了
